sdk/go/common/util/cmdutil/console.go

```diff
@@ -152,6 +153,9 @@ func (table *Table) ToStringWithGap(columnGap string) string {
 
 	allRows = append(allRows, table.Rows...)
 
+	// 7-bit C1 ANSI sequences
+	ansiEscape := regexp.MustCompile(`\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])`)
```

Comment:
I would consider making `ansiEscape` a global constant. That way it only needs to be compiled once, and not on each call.